index.html

```diff
@@ -1,21 +1,106 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="css/main.css">
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    <div class="contener">
+        <div class="text_area_nav">
+        <div class="text_nav1">
+            <div class="scouer"></div>
+
+            <h1 class="title_nav">Maya Nelson</h1>
+        </div>
+        <div class="text_nav2">
+            <h2>PROJECT MANAGER</h2>
+        </div>
+   
+    <div class="menu_bar">
+        <input id="toggle" type="checkbox"></input>
+        <label for="toggle" class="hamburger">
+            <div class="top-bun"></div>
+            <div class="meat"></div>
+            <div class="bottom-bun"></div>
+        </label>
+
+        <div class="nav">
+            <div class="nav-wrapper">
+                <nav>
+                    <a href="./index.html">ABOUT ME</a><br>
+                    <a href="./resume.html">RESUME</a><br>
+                    <a href="./projects.html">PROJECT</a><br>
+                    <a href="./contact.html">CONTACT</a>
+                </nav>
+            </div>
+        </div>
+    </div>
+</div>
+        <main>
+            <section class="info_maya">
+                <img src="img/me.png" alt="background_img" >
+                <h1 class="h1_1">Maya
+                    Nelson</h1>
+                <h1 class="underlin">______</h1>
+                <br>
```

Comment:
It is also not recommended to use the <br> tag, Read more about this in here:
https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br#:~:text=The%20main%20reason%20for%20not,space%20the%20blocks%20out%20properly.
And this comment:
https://www.quora.com/Is-br-a-web-design-best-practice